notebook_examples/notebooks_demo.ipynb

```diff
@@ -54,7 +54,7 @@
    "source": [
     "## Question #1: How does alpha diversity vary by sample group?\n",
     "\n",
-    "Here, we display Chao1, Simpson’s Index, and Shannon Entropy side-by-side, grouped by the region of birth. Each group includes samples taken across the entire three-year longitudinal study."
+    "Here, we display Observed taxa, Simpson’s Index, and Shannon Entropy side-by-side, grouped by the region of birth. Each group includes samples taken across the entire three-year longitudinal study."
```

Comment:
I'd be inclined to write "observed taxa" in lower case here, since it's not technically a proper noun? Super minor nit.

Reply:
Good nit! Fixed.